compute/instances/quickstart.mdx

```diff
@@ -5,16 +5,13 @@ meta:
 content:
   h1: Quickstart
   paragraph: This page shows you how to get started with Scaleway Elements Instances.
-steps:
-  - step: TODO
-    type: HowToStep
-    url: https://www.scaleway.com/en/docs/compute/instances/quickstart
+
 dates:
   validation: 2021-05-26
-  posted: 2021-05-26
+  posted: 2021-12-21
 ---
 
-Scaleway [Instances](/compute/instances/concepts/#instance) are computing units which provide you resources to run your applications. After you have created your Instance and installed your image of choice (e.g. an operating system), you can connect to it via SSH to use it as you wish. When you have finished with the Instance, you can delete it from your account.
+Scaleway [Instances](/compute/instances/concepts/#instance) are computing units which provide you with resources to run your applications. After you have created your Instance and installed your image of choice (e.g. an operating system), you can connect to it via SSH to use it as you wish. When you are done using the Instance, you can delete it from your account.
```

Comment:
Maybe just personal preference but I prefer the original wording (but it could be because I'm so Briiiiiitish and this sounds American ;-) )